include/OE/types/base_types.h

```diff
@@ -105,15 +105,15 @@ class CSL_WriterBase {
  * and fetching a name from an id
  * It is only supposed to be static.
  */
-class OE_Name2ID : public OE_THREAD_SAFETY_OBJECT {
+class OE_Name2ID_Container : public OE_THREAD_SAFETY_OBJECT {
```

Comment:
I made a remark that it is deprecated. It is only still used in ```types/vertex_storage.h``` and ```types/vertex_group.h```, because the way those are stored should be refactored and use a ```shared_index_map_t``` like everything else. Not yet now though.